File: ApiSuvesaPos/Negocio/Modelo/cxc/cotizaciones/cotizacion.vb
Namespace Modelo
    Public Class cotizacion
        Public Property Cotizacion As Long
        Public Property Fecha As DateTime
        Public Property Cod_Cliente As String
        Public Property Nombre_Cliente As String
        Public Property Contacto As String
        Public Property Validez As Integer
        Public Property TiempoEntrega As Integer
        Public Property Contado As Boolean
        Public Property Credito As Boolean
        Public Property Anulado As Boolean
        Public Property Dias As Integer
        Public Property Observaciones As String
        Public Property SubTotalGravada As Decimal
        Public Property SubTotalExento As Decimal
        Public Property Descuento As Decimal
        Public Property PagoImpuesto As Decimal
        Public Property Total As Decimal
        Public Property Cedula As String
        Public Property CodMoneda As Integer
        Public Property MonedaNombre As String
        Public Property SubTotal As Decimal
        Public Property Tipo_Cambio As Decimal
        Public Property Imp_Venta As Decimal
        Public Property Transporte As Decimal
        Public Property Venta As Boolean
        Public Property Exonerar As Boolean
        Public Property confirmada As Boolean
        Public Property confirmada_por As String
        Public Property EstadoActual As String
        Public Property ObservacionEstado As String
        Public Property Id_Factura As Integer
        Public Property Telefono As String
        Public Property TipoCedula As String
        Public Property CedulaCliente As String
        Public Property Cotizacion_Detalle As New Modelo.cotizacion_detalle

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/cotizaciones/cotizacion.cs
using System;

namespace Modelo
{
    public class cotizacion
    {
        public long Cotizacion { get; set; }
        public DateTime Fecha { get; set; }
        public string Cod_Cliente { get; set; }
        public string Nombre_Cliente { get; set; }
        public string Contacto { get; set; }
        public int Validez { get; set; }
        public int TiempoEntrega { get; set; }
        public bool Contado { get; set; }
        public bool Credito { get; set; }
        public bool Anulado { get; set; }
        public int Dias { get; set; }
        public string Observaciones { get; set; }
        public decimal SubTotalGravada { get; set; }
        public decimal SubTotalExento { get; set; }
        public decimal Descuento { get; set; }
        public decimal PagoImpuesto { get; set; }
        public decimal Total { get; set; }
        public string Cedula { get; set; }
        public int CodMoneda { get; set; }
        public string MonedaNombre { get; set; }
        public decimal SubTotal { get; set; }
        public decimal Tipo_Cambio { get; set; }
        public decimal Imp_Venta { get; set; }
        public decimal Transporte { get; set; }
        public bool Venta { get; set; }
        public bool Exonerar { get; set; }
        public bool confirmada { get; set; }
        public string confirmada_por { get; set; }
        public string EstadoActual { get; set; }
        public string ObservacionEstado { get; set; }
        public int Id_Factura { get; set; }
        public string Telefono { get; set; }
        public string TipoCedula { get; set; }
        public string CedulaCliente { get; set; }
        public Modelo.cotizacion_detalle Cotizacion_Detalle { get; set; } = new Modelo.cotizacion_detalle();
    }
}
